libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -35,6 +35,7 @@ const selector = 'ett-shell';
     `
       ${selector} {
         display: block;
+        width: 100%;
```

Comment:
question: Why is this necessary? The default width of a block is 100%, isn't it?

Reply:
I honestly don't remember why I added it there anymore - I do not see a reason either = Removed :-)